File: Java/백준/Bronze/20673. Covid－19/Covid－19.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int b=  in.nextInt();
        if(a<=50 && b<=10){
            System.out.println("White");
        }
        else if(b>30){
            System.out.println("Red");
        }
        else{
            System.out.println("Yellow");
        }
    }
}

File: C#/백준/Bronze/20673. Covid－19/Covid－19.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a=int.Parse(Console.ReadLine());
            int b=int.Parse(Console.ReadLine());
            if(a<=50 && b<=10){
                Console.WriteLine("White");
            }
            else if(b>30){
                Console.WriteLine("Red");
            }
            else{
                Console.WriteLine("Yellow");
            }
        }
    }
}